### ISSUE TITLE
训练结果不好，怎么调参数

### ISSUE THREAD

**Issue by 770338417lyy** [milesial/Pytorch-UNet#314]
info:validation dice coeff:0.14  左右波动，，，请问大佬些该怎么调参数

**Comment by gittjob** [comment]
用了200张图片，跑了200轮，训练集当测试集，效果如下。不知道那个环节出了问题。
![image](https://user-images.githubusercontent.com/82756842/141034945-51e01630-d98f-437b-af40-3bba7da2f091.png)

**Comment by gittjob** [comment]
不知道那个环节出了问题

**Comment by gittjob** [comment]
![image](https://user-images.githubusercontent.com/82756842/141043925-2facf20e-7e52-4c71-836b-9fb5ea1c705c.png)
The loss is strange，and the learning rate has dropped to 0

**Comment by asako98** [comment]
metoo，我觉得可能是数据集的问题

**Comment by Jazmineyjy** [comment]
> ![image](https://user-images.githubusercontent.com/82756842/141043925-2facf20e-7e52-4c71-836b-9fb5ea1c705c.png) The loss is strange，and the learning rate has dropped to 0

hello，had you solved this question?

**Comment by Ma-lele** [comment]
> ![image](https://user-images.githubusercontent.com/82756842/141043925-2facf20e-7e52-4c71-836b-9fb5ea1c705c.png) The loss is strange，and the learning rate has dropped to 0
你好请问你有啥解决办法了嘛，我的也是效果不是很好，损失降不下去

**Comment by 770338417lyy** [comment]
我换了其他的模型训练的，不好意思，没有帮助到你



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2022年11月22日(周二) 中午12:03
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [milesial/Pytorch-UNet] 训练结果不好，怎么调参数 (Issue #314)




  
 The loss is strange，and the learning rate has dropped to 0
 你好请问你有啥解决办法了嘛，我的也是效果不是很好，损失降不下去

**Comment by tensorctn** [comment]
可能是数据集差异太大，预处理不够好，模型学不到记不住，先用几张图试试能不能过拟合

**Comment by May-bit104** [comment]
> ![image](https://user-images.githubusercontent.com/82756842/141043925-2facf20e-7e52-4c71-836b-9fb5ea1c705c.png)损失诡异，学习率降至0

Hello, how did this picture come about?
